contracts/0.4.24/Lido.sol

```diff
@@ -538,140 +451,115 @@ contract Lido is ILido, StETH, AragonApp {
-            uint16 insuranceFeeBasisPoints,
-            uint16 operatorsFeeBasisPoints
-        )
-    {
-        treasuryFeeBasisPoints = uint16(TREASURY_FEE_POSITION.getStorageUint256());
-        insuranceFeeBasisPoints = uint16(INSURANCE_FEE_POSITION.getStorageUint256());
-        operatorsFeeBasisPoints = uint16(NODE_OPERATORS_FEE_POSITION.getStorageUint256());
-    }
-
-    /**
-    * @notice Returns current credentials to withdraw ETH on ETH 2.0 side after the phase 2 is launched
-    */
-    function getWithdrawalCredentials() public view returns (bytes32) {
-        return WITHDRAWAL_CREDENTIALS_POSITION.getStorageBytes32();
-    }
-
-    /**
-    * @notice Get the amount of Ether temporary buffered on this contract balance
-    * @dev Buffered balance is kept on the contract from the moment the funds are received from user
-    * until the moment they are actually sent to the official Deposit contract.
-    * @return amount of buffered funds in wei
-    */
+     * @notice Get the amount of Ether temporary buffered on this contract balance
+     * @dev Buffered balance is kept on the contract from the moment the funds are received from user
+     * until the moment they are actually sent to the official Deposit contract.
+     * @return amount of buffered funds in wei
+     */
     function getBufferedEther() external view returns (uint256) {
         return _getBufferedEther();
     }
 
     /**
-    * @notice Get total amount of execution layer rewards collected to Lido contract
-    * @dev Ether got through LidoExecutionLayerRewardsVault is kept on this contract's balance the same way
-    * as other buffered Ether is kept (until it gets deposited)
-    * @return amount of funds received as execution layer rewards (in wei)
-    */
+     * @notice Get total amount of execution layer rewards collected to Lido contract
+     * @dev Ether got through LidoExecutionLayerRewardsVault is kept on this contract's balance the same way
+     * as other buffered Ether is kept (until it gets deposited)
+     * @return amount of funds received as execution layer rewards (in wei)
+     */
     function getTotalELRewardsCollected() external view returns (uint256) {
         return TOTAL_EL_REWARDS_COLLECTED_POSITION.getStorageUint256();
     }
 
     /**
-    * @notice Get limit in basis points to amount of ETH to withdraw per LidoOracle report
-    * @return limit in basis points to amount of ETH to withdraw per LidoOracle report
-    */
+     * @notice Get limit in basis points to amount of ETH to withdraw per LidoOracle report
+     * @return limit in basis points to amount of ETH to withdraw per LidoOracle report
+     */
     function getELRewardsWithdrawalLimit() external view returns (uint256) {
         return EL_REWARDS_WITHDRAWAL_LIMIT_POSITION.getStorageUint256();
     }
 
     /**
-    * @notice Gets deposit contract handle
-    */
-    function getDepositContract() public view returns (IDepositContract) {
```

Comment:
Let's keep for backward compatibility, add deprecation warning and fetch the data from staking router